CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Unreleased
 
+## Version 2.9.13 - December 18, 2019 ##
+
+- Add `vat_number` to ShippingAddress resource [PR] (https://github.com/recurly/recurly-client-python/pull/337)
+- Remove reactivate function from Item class [PR] (https://github.com/recurly/recurly-client-python/pull/340)
```

Comment:
The space in between `[]` and `()` has caused this to be parsed improperly. Can you fix it for these and the others in this file?

Reply:
Done